samples/openapi3/client/petstore/ruby/spec/api_client_spec.rb

```diff
@@ -156,7 +156,7 @@
     end
 
     it 'fails for invalid collection format' do
-      expect(proc { api_client.build_collection_param(param, :INVALID) }).to raise_error(RuntimeError, 'unknown collection format: :INVALID')
+      expect(api_client.build_collection_param(param, :INVALID)).to raise_error(RuntimeError, 'unknown collection format: :INVALID')
```

Comment:
It should be a block:
```ruby
expect { api_client.build_collection_param(param, :INVALID) }.to
```

Reply:
I simply followed lines 155, 151, 147, etc.

Not sure why `proc` was used in the first place but looks like it doesn't work with the newer version of the dependency or ruby version (updated in the CI container)

Reply:
If wee use `raise_error`, wee need to pass a block to capture the exception.
(And 155, 151, 147, etc don't capture the exception)

> it doesn't work with the newer version of the dependency or ruby version (updated in the CI container)

Hmm... I don't know the details, but the behavior of RSpec may have changed.
In any case, it worked as expected with `expect { api_client.build_collection_param(param, :INVALID) }.to` in my environment. Can you give it a try?

Reply:
Yup, you're right that the behavior has changed somehow. Just tried it locally and using block fixes it 👍